examples/stable-diffusion/README.md

```diff
@@ -36,7 +36,8 @@ python text_to_image_generation.py \
     --use_habana \
     --use_hpu_graphs \
     --gaudi_config Habana/stable-diffusion \
-    --bf16
+    --bf16 \
+    --deterministic
```

Comment:
Please remove `--deterministic` for default recommended sample command line in README.

Reply:
Removed.